candid/evm_rpc.did

```diff
@@ -78,7 +78,7 @@ type HttpOutcallError = variant {
   };
 };
 type InitArgs = record {
-  apiKeyPrincipals : vec principal;
+  apiKeyPrincipals : opt vec principal;
```

Comment:
nit: maybe rename to something like `authorized_to_manage_api_keys` or `manage_api_keys`, the fact that it involves principals is clear from the signature.

Reply:
Sure; I'll change it to `manage_api_keys` for now (or I suppose `manageApiKeys` for the camel-case convention of the canister). Somewhat of a tricky naming situation.